website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+Other microfrontends can access shared javascript components via [cross-microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports), and apply shared, global css classes to their components in the normal way (`<div class="bold">`).
+
+Here are some examples:
+
+- https://github.com/react-microfrontends/styleguide
+- https://github.com/vue-microfrontends/styleguide
+- https://github.com/polyglot-microfrontends/styleguide
+
+The alternative to creating a utility microfrontend for your styleguide is to publish it to npm. The drawback to this approach is that it makes it easier to have duplicate copies of the styleguide, and also easier to have multiple versions of the styleguide. Npm packages are not independently deployable, nor are they singletons, but for a styleguide it's often desireable that it's centrally managed and can be deployed separately from the microfrontends that use them.
+
+### Third Party Design System
+
+When using a third-party design system, such as Material UI, Bootstrap, Semantic, etc, it is important that only one copy and version of the design system is loaded on the page. To accomplish this, here are two implementation options.
+
+1. Add the design system libraries to your SystemJS import map, then mark them as external ([full documentation](/docs/recommended-setup#sharing-with-import-maps)). Alternatively, do the equivalent with [module federation](/docs/recommended-setup#sharing-with-module-federation).
+1. Create a utility microfrontend (often called `@your-org-name/styleguide`) that contains all shared css and javascript components. Re-export the components from the design system so that all other microfrontends can access them via [cross microfrontend import](/docs/recommended-setup#cross-microfrontend-imports) (`import { Button } from '@your-org-name/styleguide';`).
+
+Once the design system is properly shared, all its CSS and javascript components will only be included one time on the web page. The code using the design system's components remains unchanged.
+
+### Global CSS versus shared Javascript components
+
+It's possible to share CSS via global CSS classes, Javascript components, or both. No method is clearly superior than the other in every way, and you should choose an approach that fits best how you'd like to do things.
+
+Some organizations scope the CSS for their shared Javascript components as a way of ensuring that the look and feel requires that you use the Javascript components. However, other organizations publish global css in addition to their javascript components, to allow for framework flexibility of their look and feel.
+
+To share Javascript components, use [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports).
+
+### CSS Variables
```

Comment:
I struggle with calling them variables. Immediately it's easier to understand than custom property, but you can easily get into trouble worry unexpected behavior when you think of them as variables.

Reply:
Changed